drivers/lora/sx12xx_common.c

```diff
@@ -276,6 +297,11 @@ int sx12xx_lora_recv(const struct device *dev, uint8_t *data, uint8_t size,
 		return ret;
 	}
 
+	if (done.result < 0) {
+		LOG_INF("Receive error");
```

Comment:
As an error this should probably be `LOG_ERR`.

Reply:
Done